tests/diagnostics/uninitialized-out-parameters.slang

```diff
@@ -3,32 +3,32 @@
 // Using before assigning
 float regular_undefined_use(out float3 v)
 {
-    //CHK-DAG: warning 41015: use of uninitialized out parameter 'v'
+    //CHK-DAG: (7): warning 41015: use of uninitialized out parameter 'v'
```

Comment:
I am AFK now but there is a better way to check the line numbers. Something like line+1 or something.
Please check the other tests for an example.